docs/typescript.md

```diff
@@ -207,6 +207,29 @@ Advanced Service should have TypeScript autocompletion.
 
 Currently, `clasp` supports [`typescript@3.3.3`](https://www.npmjs.com/package/typescript/v/3.3.3). If there is a feature in a newer TypeScript version that you'd like to support, or some experimental flag you'd like enabled, please open an issue.
 
+#### TypeScript configuration
+
+TypeScript configuration support is currently limited as below:
+
+- The configuration file must be named `tsconfig.json` and located in the same folder as the `.clasp.json` project files.
```

Comment:
You can create a TypeScript configuration file by creating a `tsconfig.json` file in the same folder as your `.clasp.json` file.